wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,791 @@
+# Formatting
+
+*For the writing standards, see: [Article style criteria/Writing](../Writing)*
+
+*Notice: This article uses [RFC 2119](https://tools.ietf.org/html/rfc2119 "IETF Tools") to describe requirement levels.*
+
+## Locales
+
+Listed below are the properly-supported locales for the wiki:
+
+| File Name | Locale Name | Native Script |
+| :-- | :-- | :-- |
+| `en.md` | English | English |
+| `ar.md` | Arabic | اَلْعَرَبِيَّةُ |
+| `be.md` | Belarusian | Беларуская мова |
+| `bg.md` | Bulgarian | Български |
+| `cs.md` | Czech | Česky |
+| `da.md` | Danish | Dansk |
+| `de.md` | German | Deutsch |
+| `gr.md` | Greek | Ελληνικά |
+| `es.md` | Spanish | Español |
+| `fi.md` | Finnish | Suomi |
+| `fr.md` | French | Français |
+| `hu.md` | Hungarian | Magyar |
+| `id.md` | Indonesian | Bahasa Indonesia |
+| `it.md` | Italian | Italiano |
+| `ja.md` | Japanese | 日本語 |
+| `ko.md` | Korean | 한국어 |
+| `nl.md` | Dutch | Nederlands |
+| `no.md` | Norwegian | Norsk |
+| `pl.md` | Polish | Polski |
+| `pt.md` | Portuguese | Português |
+| `pt-br.md` | Brazilian Portuguese | Português (Brasil) |
+| `ro.md` | Romanian | Română |
+| `ru.md` | Russian | Русский |
+| `sk.md` | Slovak | Slovenčina |
+| `sv.md` | Swedish | Svenska |
+| `th.md` | Thai | ไทย |
+| `tr.md` | Turkish | Türkçe |
+| `uk.md` | Ukrainian | Українська мова |
+| `vi.md` | Vietnamese | Tiếng Việt |
+| `zh.md` | Chinese (Simplified) | 简体中文 |
+| `zh-tw.md` | Traditional Chinese (Taiwan) | 繁體中文（台灣） |
+
+*Note: The website will give readers their selected language's version of an article. If it is not available, the English version will be given.*
+
+If a language is not listed above, append `?locale={langcode}` to the URL (e.g. `https://osu.ppy.sh/help/wiki/Welcome?locale=zh`). You can also help translate the [osu-web via Crowdin](https://crowdin.com/project/osu-web). If you need help with translating or want to discuss translating decisions, discuss in the [osu!dev Discord](https://discord.gg/ppy) (`#osu-web` channel).
+
+### Content parity
+
+Translations are subject to strict content parity with their English article, in the sense that they must have the same message, regardless of grammar and syntax. Any changes to the translations' meanings must be accompanied by equivalent changes to the English article.
+
+There are some cases where the content is allowed to differ:
```

Comment:
I'd like to add subcommunity-specific explanations, such as [this paragraph in Support Team](https://github.com/ppy/osu-wiki/blob/master/wiki/People/The_Team/Support_Team/ru.md#L19), which says `If you need help, open a thread in one of these forums, and you'll be contacted. If you need help in Russian, there's a dedicated thread in the Russian subforum called «...».`